tools/frr-reload.py

```diff
@@ -2170,6 +2185,18 @@ def compare_context_objects(newconf, running):
                     if x == 1 and ctx_keys[0].startswith("no "):
                         continue
 
+                    depth = len(ctx_keys)
+                    if not line:
+                        depth -= 1
+
+                    # If current line is a single-line context command
+                    # prepend it with proper number of 'exit's from
+                    # previous context. This prevents global 'bgp graceful-shutdown'
+                    # from running in a local 'router bgp ...' context.
```

Comment:
This should add `exit` statements for lots of scenarios that don't involve `bgp graceful-shutdown`, right? If so, please remove this comment.